ns1/resource_zone.go

```diff
@@ -120,6 +121,14 @@ func resourceZone() *schema.Resource {
 				Optional: true,
 				Computed: true,
 			},
+			"create_ns_record": {
```

Comment:
we were talking about what to call this key, and were interested to see what you came up with :)

best stab i had was `autogenerate_ns_record`, I think it's clearer